lisa/sut_orchestrator/util/resize_partition.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+
+from lisa.node import RemoteNode
+from lisa.tools import Mount
+from lisa.util import LisaException
+
+
+def expand_root_partition(node: RemoteNode) -> None:
```

Comment:
Can you put it in a tool called `ResizePartition`? And there is another similar method `_expand_os_partition` in `microsoft\testsuites\power\common.py`

Reply:
Done